### DISCUSSION TITLE
Does Keygen support Dongles?

### DISCUSSION THREAD

**Discussion by chcuongdsi** [keygen-sh/community#154] [General]
We are using flexera flexnet and it supports Dongle. We store the license file in Dongle so customers can use it as an offline license

**Comment by ezekg** [comment]
There's nothing stopping you from loading [a license file](https://keygen.sh/docs/api/cryptography/#cryptographic-lic) onto a USB drive and distributing it to your customers, and then having your software look for and automatically load the license file from that type of USB drive.

Do you have a more specific question here? More specific requirements?
